ports/mimxrt/boards/OLIMEX_RT1010/board.json

```diff
@@ -12,8 +12,8 @@
         "OLIMEX_RT1010Py.jpg"
     ],
     "mcu": "mimxrt",
-    "product": "Olimex_RT1010Py",
+    "product": "RT1010-Py",
     "thumbnail": "",
-    "url": "https://www.olimex.com/Products/ARM/NXP",
-    "vendor": "OLIMEX"
+    "url": "https://www.olimex.com/Products/MicroPython/RT1010-Py",
+    "vendor": "Olimex"
```

Comment:
Stay all-caps?

Reply:
Olimex use their name in a mixture of upper and Pascal case (like on their [about page](https://www.olimex.com/About/)). [Mouser](https://au.mouser.com/manufacturer/olimex/) and [Digikey](https://www.digikey.com.au/en/supplier-centers/olimex) list them using Pascal case.

I find the upper case visually ugly so prefer the Pascal case but can use upper if that's your preference?

Reply:
You're right, it is a bit inconsistent.

It looks like this is their business registration: https://portal.registryagency.bg/CR/en/Reports/ActiveConditionTabResult?uic=825241718 and since it's Bulgarian there's no real English source of truth for the name, except how they themselves translate it.

Os, let's go with "Olimex".